composer.json

```diff
@@ -15,7 +15,7 @@
     },
     "require-dev": {
         "mindplay/benchpress": "dev-master",
-        "mindplay/testies": "^0.2",
+        "mindplay/testies": "dev-master",
```

Comment:
Do you really need to depend on dev-master? Wouldn't it be nicer to release a new version of testies perhaps?